yarn.lock

```diff
@@ -6692,10 +6692,9 @@ lolex@^3.1.0:
   version "3.1.0"
   resolved "https://registry.yarnpkg.com/lolex/-/lolex-3.1.0.tgz#1a7feb2fefd75b3e3a7f79f0e110d9476e294434"
 
-loom-js@^1.49.4:
+"loom-js@https://github.com/loomnetwork/loom-js#dist-dpos-v3":
```

Comment:
Temporarily using an open PR's branch, https://github.com/loomnetwork/loom-js/pull/263, with `dist-dpos-v3` having the build files.